packages/yoroi-extension/app/components/profile/terms-of-use/TermsOfUseText.scss

```diff
@@ -27,38 +28,60 @@
     margin-left: 20px;
   }
 
-  p, li {
+  p,
+  li {
     font-weight: 400;
     font-size: 14px;
     letter-spacing: 0.5px;
     line-height: 1.38;
     margin-bottom: 11px;
-
   }
 }
 
 :global(.YoroiClassic) .terms {
   color: var(--yoroi-terms-of-use-text-color);
 }
 
-:global(.YoroiModern), :global(.YoroiRevamp) {
+:global(.YoroiModern),
+:global(.YoroiRevamp) {
   .terms {
-    h1, h2 {
+    h1,
+    h2 {
       font-size: 16px;
     }
-    
+
     h1 {
       margin-top: 26px;
     }
-    
+
     h2 {
       margin-top: 46px;
     }
-    
-    p, li {
+
+    p,
+    li {
       color: var(--yoroi-terms-of-use-text-color);
       font-size: 13px;
       line-height: 1.54;
     }
   }
 }
+
+:global(.YoroiRevamp) .terms {
```

Comment:
We shouldn't be using plain css for the screens (at least for the conditions applied for the revamp theme), because when we'll only use revamp (after the flip) the DS and theming will cause issues.